packages/strapi-plugin-i18n/controllers/content-types.js

```diff
@@ -50,7 +53,9 @@ module.exports = {
 
     ctx.body = {
       nonLocalizedFields: sanitizedNonLocalizedFields,
```

Comment:
unrelated but that would have been more consistent to call that nonoLocalizedAttributes

Reply:
Yes, but the route is called `fields` too. We started 2 different names and should refactor everywhere